website/client-old/js/config.js

```diff
@@ -35,7 +35,7 @@ angular.module('habitrpg')
           if (isUserLoaded && notification.type === 'CRON') {
             // If the user is already loaded, do not show the notification, syncing will show it
             // (the user will be synced automatically)
-            $rootScope.User.readNotification(notification.id);
+            // $rootScope.User.readNotifications([notification.id]);
```

Comment:
Well, CRON has tests to combine notifications. And, this breaks the code by creating a race condition of sorts. So, I'm not sure what id does or if it is needed from tests.